packages/gnome_session.rb

```diff
@@ -37,17 +37,9 @@ class Gnome_session < Package
   depends_on 'libxcomposite' # R
   depends_on 'mesa' # R
 
-  def self.build
-    system "meson setup #{CREW_MESON_OPTIONS}\
-      -Dsystemd=false \
-      -Dsystemd_session=disable \
-      -Dsystemd_journal=false \
-      builddir"
-    system 'meson configure --no-pager builddir'
-    system 'ninja -C builddir'
-  end
+  gnome
 
-  def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
-  end
+  meson_options '-Dsystemd=false \
```

Comment:
Do we need the additional `-Dsystemd_session=disable` and `-Dsystemd_journal=false` arguments if we already pass `-Dsystemd=false`?